src/core/shape/attributes.js

```diff
@@ -312,39 +538,57 @@ p5.prototype.strokeJoin = function(join) {
 };
 
 /**
- * Sets the width of the stroke used for lines, points, and the border around
- * shapes. All widths are set in units of pixels.
+ * Sets the width of the stroke used for points, lines, and the border around
```

Comment:
This is related to the 'stroke()' reference and the [comment](https://github.com/processing/p5.js/pull/6839#discussion_r1518993692) in #6839 

Is it important to keep both 'lines' and 'border'? If so, we should keep the consistency to referring border, which is currently called 'outline' in stroke() reference. 
@limzykenneth @davepagurek

Reply:
I think changing `border` to `outline` here makes sense! I'm also in favour of keeping points and lines to make clear what this gets used for.

Reply:
Updated!